gemd/demo/cake.py

```diff
@@ -30,6 +30,27 @@
 TEMPLATE_SCOPE = DEMO_SCOPE + '-template'
 
 
+def change_scope(data, *, templates=None):
```

Comment:
Dynamically changing a global variable seems risky. Why do you need to do this?

Reply:
This is a minimal change to support having multiple copies of the dataset be able to co-exist on platform.  If the e2e tests are run with CITRINE-NO-PRELOAD=1, for example, this let's us have these tests still meaningful.  This, for example, would have prevented the issues plaguing [nextgen pull 622](https://github.com/CitrineInformatics/nextgen-devkit/pull/622).

Reply:
Gotcha 👍

Reply:
I agree with @ventura-rivera : wrapping this whole module in a class with scope as one of the class members would be preferable.  We could do that in a fast-follow on though to keep the changes in this PR tighter.

Reply:
Wrapping in a class could make sense, but would require generation of a singleton in order to avoid breaking downstream code.